MonoGame.Framework/Color.cs

```diff
@@ -296,7 +296,7 @@ public Color(float r, float g, float b, float alpha)
         }
 
         /// <summary>
-        /// Gets or sets the blue component of <see cref="Color"/>.
+        /// Gets or sets the blue component of this <see cref="Color"/>.

```

Comment:
I really dislike this pattern of referencing the class itself from every member.  

Like if you can see "this Color" "this Color" "this Color" "this Color" "this Color" repeated in every member that means the information is redundant.

Saying...
```
/// Gets or sets the blue component.
```
... is enough.  The user understands it is the blue component of "this Color".

Just noticed this appearing a lot and i'd like to see us stop doing it.